blackbox/docs/general/information-schema.rst

```diff
@@ -235,6 +235,8 @@ options of all available views.
 +---------------------+-------------------------------------------------------------------------------------+-------------+
 | ``is_updatable``    | Whether the view is updatable. Not applicable for CrateDB, always returns ``FALSE`` | ``Boolean`` |
 +---------------------+-------------------------------------------------------------------------------------+-------------+
+| ``owner``           | The user that created the view                                                      | ``String``  |
```

Comment:
Should we call this `definer` (https://dev.mysql.com/doc/refman/5.7/en/views-table.html) ?

Reply:
I don't mind calling it `owner`, because it's not standardized. Also don't have a strong opinion.